diff --git a/node_modules/@capacitor/cli/dist/ios/update.js b/node_modules/@capacitor/cli/dist/ios/update.js
index 4d87b7b..b9794f8 100644
--- a/node_modules/@capacitor/cli/dist/ios/update.js
+++ b/node_modules/@capacitor/cli/dist/ios/update.js
@@ -93,7 +93,9 @@ let package = Package(
         )
     ]
 )`;
-    await (0, fs_extra_1.writeFile)((0, path_1.join)(config.ios.cordovaPluginsDirAbs, 'sources', p.name, 'Package.swift'), content);
+    const pluginSourceDir = (0, path_1.join)(config.ios.cordovaPluginsDirAbs, 'sources', p.name);
+    await (0, fs_extra_1.ensureDir)(pluginSourceDir);
+    await (0, fs_extra_1.writeFile)((0, path_1.join)(pluginSourceDir, 'Package.swift'), content);
 }
 async function installCocoaPodsPlugins(config, plugins, deployment) {
     await (0, common_1.runTask)(`Updating iOS native dependencies with ${colors_1.default.input(`${await config.ios.podPath} install`)}`, () => {
